src/ai/ai_help_structs.cc

```diff
@@ -802,6 +824,7 @@ void ManagementData::mutate(const uint8_t pn) {
 					const int16_t new_value = shift_weight_value(
 					   get_military_number_at(i), mutating_intensity == MutatingIntensity::kAgressive);
 					set_military_number_at(i, new_value);
+					mutation_stat[kMilitaryNumbersPos] += 1;
```

Comment:
our codechecker has become very picky. its complaining now to use ++mutation_stat[kMilitaryNumbersPos] instead of += 1 
same for lines 860 and 901

Reply:
fixed